views/standard/spec/columns.js

```diff
@@ -293,6 +293,13 @@ define(['jquery', 'appearanceUtils', 'underscore'], function($, utils, _) {
                    label.created_at, label.user);
                   node.append(icon);
                }
+               if ((label = pull.getLabel('external_block'))) {
```

Comment:
Is this supposed to be an assignment or a comparison?

Reply:
Assignment. It's using attributes of `label` a little lower down.

Reply:
Assignment. Well, sort of both, but the single equals is right here. `label` gets assigned the result from  `pull.getLabel()`. If the label is `external_block` then the statement will evaluate to true, and you'll enter the if.

Reply:
It looks like there's an extra set of parens. Is that intentional?

Reply:
I just stayed consistent with the if for the QAing label. I think the parens could probably be removed. Want me to take them off both?